jpsonic-main/src/main/java/com/tesshu/jpsonic/service/upnp/processor/AlbumUpnpProcessor.java

```diff
@@ -57,49 +56,34 @@ public ProcId getProcId() {
         return ProcId.ALBUM;
     }
 
-    @Override
-    public BrowseResult browseRoot(String filter, long firstResult, long maxResults) throws ExecutionException {
-        DIDLContent didl = new DIDLContent();
-        List<Album> selectedItems = albumDao.getAlphabeticalAlbums((int) firstResult, (int) maxResults, false, true,
-                util.getGuestFolders());
-        for (Album item : selectedItems) {
-            addItem(didl, item);
-        }
-        return createBrowseResult(didl, (int) didl.getCount(), getDirectChildrenCount());
-    }
-
     @Override
     public Container createContainer(Album album) {
         return factory.toAlbum(album);
     }
 
-    @Override
-    public int getDirectChildrenCount() {
-        return albumDao.getAlbumCount(util.getGuestFolders());
-    }
-
     @Override
     public List<Album> getDirectChildren(long offset, long maxResults) {
         return albumDao.getAlphabeticalAlbums((int) offset, (int) maxResults, false, true, util.getGuestFolders());
     }
```

Comment:
Confirm that it is Alphabetical only here. (Other Album view will be branched to Alphabetical or Year depending on settings)

Reply:
No problem.

[Alphabetical]
AlbumUpnpProcessor

[Alphabetical/Year]
ArtistUpnpProcessor
ArtistByFolderUpnpProcessor
MediaFileUpnpProcessor
AlbumByGenreUpnpProcessor -> MediaFileUpnpProcessor
IndexUpnpProcessor -> MediaFileUpnpProcessor

[Random]
RandomAlbum

[Created]
RecentAlbumId3UpnpProcessor
RecentAlbumUpnpProcessor